### PR TITLE
Fix minor initializing warning in default ctor of wxRibbonBar.

### PR DESCRIPTION
**Pull request by Runtemund**
Just a minor fix of missed member initialization.

### CODE REVIEW

File: src/ribbon/bar.cpp [hunk 1]
```diff
@@ -733,7 +733,9 @@ wxRibbonBar::wxRibbonBar()
     m_tab_scroll_buttons_shown = false;
     m_arePanelsShown = true;
     m_help_button_hovered = false;
-
+    m_bar_hovered = false;
+    m_toggle_button_hovered = false;
+    m_ribbon_state = wxRIBBON_BAR_PINNED;
```
Comment (vadz) [inline src/ribbon/bar.cpp:738]: I think this makes the same lines in `CommonInit()` redundant, doesn't it? I.e. they should be deleted from there now, right?

Comment (Runtemund) [reply src/ribbon/bar.cpp:738]: This was just for suppressing compiler warnings under W10. Not more...

Comment (vadz) [reply src/ribbon/bar.cpp:738]: What were the warnings (and with which compiler) exactly? I thought it was to make sure these fields are initialized, and this change seems correct to me, but it also makes initializing them in the other place unnecessary AFAICS.

Comment (Runtemund) [reply src/ribbon/bar.cpp:738]: It is C26495 under Visual Studio Community 2022. But it seems that it is just a hint from IntelliSense, not the compiler itself. So... commit or drop. It's up to you.

Comment (vadz) [reply src/ribbon/bar.cpp:738]: Could you please confirm that you don't get this analyser warning if you add these lines and remove them from `CommonInit()`? If yes, please update this PR and I'll merge it. TIA!

**Pull request closed without merging**